getid3/module.graphic.bmp.php

```diff
@@ -633,15 +634,15 @@ public function PlotBMP(&$BMPinfo) {
 			return false;
 		}
 		set_time_limit(intval(round($BMPinfo['resolution_x'] * $BMPinfo['resolution_y'] / 10000)));
-		if ($im = ImageCreateTrueColor($BMPinfo['resolution_x'], $BMPinfo['resolution_y'])) {
+		if ($im = imagecreatetruecolor($BMPinfo['resolution_x'], $BMPinfo['resolution_y'])) {
```

Comment:
Call to function imagecreatetruecolor() with incorrect case: ImageCreateTrueColor